checker.cs

```diff
@@ -3,32 +3,81 @@
 
 class Checker
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
+    public static string isBPMok(float bpm)
+    {
+        string warning = "current bpm: "+bpm+ ",bpm normal";
+        if(bpm < 70)
+        {
+            warning = "current bpm: "+bpm+ ",bpm too low";
         }
-        return true;
+        else if(bpm > 150)
+        {
+            warning = "current bpm: "+bpm+ ",bpm too high";
+        }
+        return warning;
     }
-    static void ExpectTrue(bool expression) {
-        if(!expression) {
-            Console.WriteLine("Expected true, but got false");
-            Environment.Exit(1);
+
+    public static string isSPO2ok(float spo2)
+    {
+        string warning = "current spo2: "+spo2+ ",spo2 normal";
+        if(spo2 < 90)
+        {
+            warning = "current spo2: "+spo2+ ",spo2 too low";
         }
+        else if(spo2 > 100)
+        {
+            warning = "current spo2: "+spo2+ ",spo2 too high";
+        }
+        return warning;
     }
-    static void ExpectFalse(bool expression) {
-        if(expression) {
-            Console.WriteLine("Expected false, but got true");
-            Environment.Exit(1);
+
+    public static string isRespRateok(float respRate)
+    {
+        string warning = "current respRate: "+respRate+ ",respRate normal";
+        if(respRate < 30)
+        {
+            warning = "current respRate: "+respRate+ ",respRate too low";
         }
+        else if(respRate > 95)
+        {
+            warning = "current respRate: "+respRate+ ",respRate too high";
+        }
+        return warning;
+    }    
+
+    static void areVitalsOk(float bpm, float spo2, float respRate)
+    {
+        Console.WriteLine(isBPMok(bpm));
+        Console.WriteLine(isSPO2ok(spo2));
+        Console.WriteLine(isRespRateok(respRate));
     }
+
+    // static void ExpectTrue(bool expression) {
```

Comment:
Do not have commented code in commits. If you need old code it will be in the older commits.